src/flask-api/net_predictor/WR_logistic_regression.py

```diff
@@ -0,0 +1,119 @@
+
+  combined_stats_train = pd.merge(team_win_loss_train, opponent_stats_train, on=['Opponent Name'])
+  combined_stats_train[team_stats_train.columns] = team_stats_train.iloc[0]
+  # combined_stats_test = pd.merge(team_win_loss_test, opponent_stats_test, on=['Opponent Name'])
+  # combined_stats_test[team_stats_test.columns] = team_stats_test.iloc[0]
+
+  complete_data_train = complete_data_train.append(combined_stats_train)
+  # complete_data_test.append(combined_stats_test)
+
+for team_name in valid_teams_test:
+  team_win_loss_test = kpteam.get_schedule(browser, team=team_name, season=testing_season)
+
+  # Get a teams schedule by getting a dataframe with whether or not they won or lost, and the name of their opponent
+  team_win_loss_test = team_win_loss_test[team_win_loss_test['Opponent Name'].isin(valid_teams_test)][['Result', 'Opponent Name', 'Location']]
+  team_win_loss_test['Result'] = team_win_loss_test['Result'].apply(lambda x: x[:1])
+  team_win_loss_test['Result'] = team_win_loss_test['Result'].replace({'W': 1, 'L': 0})
+  team_win_loss_test = pd.get_dummies(team_win_loss_test, columns=['Location']) # One-Hot Encoding
+
+  team_stats_test = stats_test[stats_test['Team'] == team_name].drop(['Conference', 'Team'], axis=1)
+  team_stats_test.columns = 'team_' + team_stats_test.columns[0:]
+
+  # Rename opponent stats to have the prefix 'oponnent_'
+  opponent_stats_test = stats_test.drop(['Conference'], axis=1)
+  opponent_stats_test.columns = opponent_stats_test.columns[:1].union('opponent_' + opponent_stats_test.columns[1:])
+  opponent_stats_test = opponent_stats_test.rename(columns={'Team': 'Opponent Name'})
+
+  combined_stats_test = pd.merge(team_win_loss_test, opponent_stats_test, on=['Opponent Name'])
+  combined_stats_test[team_stats_test.columns] = team_stats_test.iloc[0]
+
+  complete_data_test = complete_data_test.append(combined_stats_test)
+
+complete_data_train = complete_data_train.fillna(0)
+complete_data_test = complete_data_test.fillna(0)
+
+print(complete_data_train)
+print(complete_data_test)
+
+# Get the names of the features based on columns_to_drop
+feature_names = complete_data_train.drop(columns_to_drop, axis=1).columns
+print(feature_names)
+
+x_train = complete_data_train[feature_names].values.astype(float)
+y_train = complete_data_train[label_name].values.reshape(-1,1).astype(float)
+x_test = complete_data_test[feature_names].values.astype(float)
+y_test = complete_data_test[label_name].values.reshape(-1,1).astype(float)
+
+# Fit and predict with the model
+model = LogisticRegression()
+model.fit(x_train, y_train)
+y_train_pred = model.predict(x_train)
+y_test_pred = model.predict(x_test)
+
+# Determine accuracy using r2 score
+print('Sklearn\'s model\'s train and test accuracy')
+print("Training Accuracy:", metrics.accuracy_score(y_train, y_train_pred))
+print("Testing Accuracy:", metrics.accuracy_score(y_test, y_test_pred))
```

Comment:
These are great!  Maybe we should return them so that we can call this a few times in our api.